src/profile.c

```diff
@@ -1282,10 +1282,10 @@ profile_pass_rewrite_sid_set (void *in, const void *v)
   const uint16_t *val = v;
   if (*val != pro->pro_rewrite_sid) {
     if (*val > 0) {
-      pro->pro_rewrite_pmt =
-      pro->pro_rewrite_pat =
-      pro->pro_rewrite_sdt =
-      pro->pro_rewrite_nit =
+      pro->pro_rewrite_pmt = 1;
+      pro->pro_rewrite_pat = 1;
+      pro->pro_rewrite_sdt = 1;
+      pro->pro_rewrite_nit = 1;
       pro->pro_rewrite_eit = 1;
```

Comment:
Since my changes on src/profile.c are already in PR #1400: should I remove this commit from this PR?

Reply:
Yes it should be removed from this PR.

If I see it correctly, the only change necessary is changing the else to an if?

Reply:
Yes, in principle changing the else to an if is the only change necessary. The other changes in the file `muxer_pass.c` are just rewriting the pmt variable to make it look a little bit more consistent.
I managed to get rid of these changes by rebasing my branch. Unfortunately, I now ended up having the changes of `README.md` in the pull request again. :-(
I am somehow still struggling with GitHub and I see the only solution in killing this pull request, branching off freshly, do the change by hand and start a new pull request. @Flole998 , do you have a better suggestion?